app/src/main/java/com/github/se/assocify/model/entities/Task.kt

```diff
@@ -9,16 +11,16 @@ package com.github.se.assocify.model.entities
  * @param isCompleted whether the task is completed
  * @param startTime time when the task should start
  * @param peopleNeeded number of people needed to complete the task
- * @param category category of the task
+ * @param type category of the task
```

Comment:
alright nevermind category is better...
```suggestion
 * @param category what the task is for (catering, committee...)
```